Nodejs/Product/Nodejs/TestFrameworks/Tape/tape.js

```diff
@@ -0,0 +1,75 @@
+var fs = require('fs');
```

Comment:
[nit] I'm not sure our other test frameworks do this, but please add `"use strict";` for this new file.